dockstore-webservice/src/main/resources/migrations.1.14.0.xml

```diff
@@ -294,4 +294,10 @@
             CREATE UNIQUE INDEX case_insensitive_workflow_workflowname on workflow using btree (sourcecontrol, organization, repository, lower(workflowname)) where workflowname is not null and id > 18876;
         </sql>
     </changeSet>
+    <changeSet id="reset open data" author="coverbeck">
+        <update tableName="version_metadata">
+            <!--  Not explicitly specifying a value sets it to null-->
+            <column name="publicaccessibletestparameterfile"></column>
```

Comment:
Ideally we would rename this column and the corresponding property as it now means "everything needed to run the workflow is on Dockstore", i.e., no file inputs, or file inputs and a test parameter file with all public access data. But renaming properties is a hassle if we want the UI to work with no downtime and the CLI not to break. So I'm inclined to just leave it.

Reply:
Not too worried if there is a good comment